ocs_ci/deployment/tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf8 -*-
+
+import json
+
+import pytest
+
+from ocs_ci.framework import config
+
+
+@pytest.fixture
+def clusterdir(request, tmpdir):
+    """
+    This fixture creates an openshift cluster dir with minimal metadata.json
+    file inside so that ocs-ci deployment class can be instantiated.
+    """
+    # TODO: Do we want to add `cluster_path = null` to ENV_DATA section
+    #       of ocs_ci/framework/conf/default_config.yaml file?
+    # we assume that cluster_path is not preconfigured
```

Comment:
We do this for `cluster_name` so I don't think it will be an issue. However, `--cluster-path` is a [required CLI param](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/pytest_customization/ocscilib.py#L311-L313) so we should never really hit the `null`.

Reply:
I note it here because it has implications for the unit tests there. If `ENV_DATA` doesn't have `cluster_path` key, instantiation of a deployment class fails on this very fact. So the note is basically saying that later, it would be nice to figure out whether is deliberate decision or not.

Reply:
I could have tried to add `cluster_path = null` to `ocs_ci/framework/conf/default_config.yaml`, but I would have to check what implications such change have, moreover I tried not to change unrelated code too much. So I implemented the required setup and teardown in a unit test instead, which captures the current behavior.